packages/project-utils/bundling/function/telemetryFunction.js

```diff
@@ -1,7 +1,7 @@
 const userFunction = require("./_handler.js");
 const https = require("https");
 
-const TELEMETRY_ENDPOINT = "d16ix00y8ek390.cloudfront.net";
+const TELEMETRY_ENDPOINT = "d1hyim4iinlmte.cloudfront.net";
```

Comment:
This is not the correct URL, this is the URL over which the WCP React app is being served. What we actually need is the URL over which the backend is served (GraphQL API, telemetry endpoint, etc.).

Let's do three things here:

1. Let's actually use the backend's CloudFront URL
2. Let's add a comment above it, so that the developer knows what this URL represents. With this cryptic CF URL, it's not obvious what it represents.
3. Please update the Notion doc / update PROD / WCP / GraphQL API URL (https://www.notion.so/webiny/WCP-URLs-ae0b3f276fe94532a1ad2c8b38ef4987)
4. Let's be a bit more specific with the naming here. The URL does not represent the telemetry endpoint, it's actually the domain name of the backend's CloudFront Distribution. So, I think something like API_DOMAIN_NAME would be more appropriate. Alternatively, it might also make more sense to keep the `TELEMETRY_ENDPOINT` or `TELEMETRY_URL`, but then change this value to a full URL, so `https://xxx.cloudfront.net/telemetry`.
5. Below, let's use `node-fetch` instead of the low level `https` lib. It will make things simpler.